### PR TITLE
npm: Fixed `test` script

### PR DESCRIPTION
**Pull request by RunDevelopment**
Because of #1646 the command `mocha tests/run.js` was no longer the last command in the chain which means that arguments like `npm test -- --language=css` were not passed to it.
This fixes the issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "prism.js",
   "style": "themes/prism.css",
   "scripts": {
-    "test": "mocha tests/testrunner-tests.js && mocha tests/run.js && mocha tests/plugins/**/*.js"
+    "test": "mocha tests/testrunner-tests.js && mocha tests/plugins/**/*.js && mocha tests/run.js"
```
Comment (mAAdhaTTah) [inline package.json]: We should split this into 3 commands, and have `test` run them, so if someone wants to test the languages, they could do `npm run test:languages -- --language=css` without issue. It also allows you to e.g. just test plugins without needing to run the other tests around it.

**Comment by RunDevelopment** [comment]
Like this @mAAdhaTTah ?
This, of course, has the disadvantage that none of the arguments of `npm test` are passed to either of the called commands.

**Comment by mAAdhaTTah** [comment]
@RunDevelopment Yeah, that's what I was thinking. If you're just testing a single language, you wouldn't run the main command, you'd just run `npm run test:languages` with the appropriate flag. You wouldn't want to run all the other tests at the same time.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,10 @@
   "main": "prism.js",
   "style": "themes/prism.css",
   "scripts": {
-    "test": "mocha tests/testrunner-tests.js && mocha tests/run.js && mocha tests/plugins/**/*.js"
+    "test:tests": "mocha tests/testrunner-tests.js",
```
Comment (mAAdhaTTah) [inline package.json]:
```suggestion
    "test:runner": "mocha tests/testrunner-tests.js",
```

**Comment by RunDevelopment** [comment]
Alright, then I'll adjust the docs and then this should be gtg.

**Pull request merged**